templates/repo/diff/compare.tmpl

```diff
@@ -192,7 +192,7 @@
 						<span class="index">#{{.PullRequest.Issue.Index}}</span>
 					</div>
 				</div>
-				<a href="{{$.RepoLink}}/pulls/{{.PullRequest.Issue.Index}}" class="ui compact button primary">
+				<a role="button" href="{{$.RepoLink}}/pulls/{{.PullRequest.Issue.Index}}" class="ui compact button primary">
```

Comment:
I believe it is a link but not a button.

A button means it do something on current page, a link means it navigates to another place.

Here, it navigates, so it can't be a button.

Reply:
It renders like a button, but whatever, I can revert.